### PR TITLE
Textarea - Fix inconsistencies of disable field style

### PR DESCRIPTION
**Pull request by ericangeles**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Fixes the inconsistencies of the disable field styles on `Textarea`, `Dropdown`, `Colorpicker`, `Multi-Select`, `Lookup`, `Datepicker`, `Checkbox`, `Autocomplete`, `Editor`, `Timepicker`, `Spinbox`, `Radio`, `Slider`, `Switch`, `Searchfield` and `Input` components.

Updated it by creating a single test page for all the components that will be disable.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/3917

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Navigate to http://localhost:4000/components/form/example-disabled-field.html
- Try to play around all the components
- Hit `Click to Disable` button
- All the components should be disabled
- Hit `Click to Enable` button
- Enable function should also work properly
- Test it on different themes and variants combination

**Included in this Pull Request**:
- [x] A note to the change log.

### CODE REVIEW

**Comment by ericangeles** [comment]
Some test might fail, but will fix it once the CI done.

**Comment by ericangeles** [comment]
If it's passed on CI, this is ready for review.

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +30,7 @@
 - `[Popupmenu]` Fixed a bug that the aria items are in the wrong place. Its now using [this guide](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html). ([#4085](https://github.com/infor-design/enterprise/issues/4085))
 - `[Popupmenu]` Fixed a bug where the heading doesn't display properly with multi-select menu. ([#3926](https://github.com/infor-design/enterprise/issues/3926))
 - `[Searchfield]` Fixed an issue where some of the searchfield examples did not have focus states. ([#1060](https://github.com/infor-design/enterprise/issues/1060))
+- `[Textarea]` Fixed inconsistencies on styling of disabled field when using disable function. ([#3917](https://github.com/infor-design/enterprise/issues/3917))
```
Comment (tmcconechy) [inline docs/CHANGELOG.md:33]:
```suggestion
- `[Textarea]` Fixed inconsistencies on styling of disabled field when using disable function, now the label will disable on all components when using this function. In general the label should be dimmed on disabled fields as per the design. ([#3917](https://github.com/infor-design/enterprise/issues/3917))
```

File: src/components/editor/editor.js [hunk 2]
```diff
@@ -2925,6 +2925,10 @@ Editor.prototype = {
     this.destroyPreviewMode();
     this.element.addClass('is-disabled').attr('contenteditable', 'false');
     this.container.addClass('is-disabled');
+
+    setTimeout(() => {
```
Comment (tmcconechy) [inline src/components/editor/editor.js:2929]:
```suggestion
    setTimeout(() => {
```

Is this timeout really needed?

Comment (ericangeles) [reply src/components/editor/editor.js:2929]: When I invoked the disable function, it doesn't work. I suspected that it calls before when you really needed it.

**Comment by ericangeles** [comment]
It's weird that the colorpicker test baseline still failing after making some changes. Verified this also in local and it passed. Not sure what's wrong. @tmcconechy

**Comment by tmcconechy** [comment]
I saw it fail too on another PR this is the diff https://s3.amazonaws.com/hl-travisci-artifacts/infor-design/enterprise/16740/16740.3/test/.tmp/diff/colorpicker-open-chrome-1200x800-dpr-1.png

Im rerunning it now lets see https://travis-ci.com/github/infor-design/enterprise/jobs/361003633

**Comment by tmcconechy** [comment]
Also not sure why but i merged master on to fix the color picker test and this is the ONLY branch it didnt fix. I saw you did something with the test as well. Maybe you need to re make the color picker test again on chrome 84 or delete and resync the color picker test image with master?

**Comment by ericangeles** [comment]
Ready to review now again, @tmcconechy @EdwardCoyle. Thanks for the feedback 💪

**Pull request merged**